core/services/ping/ping1d_mavlink.py

```diff
@@ -55,7 +55,7 @@ def distance_message(time_boot_ms: int, distance_cm: int, device_id: int, confid
             "horizontal_fov": 0.52,
             "vertical_fov": 0.52,
             "quaternion": [0, 0, 0, 0],
-            "signal_quality": confidence,
+            "signal_quality": min(1, confidence),  # 0 means undefined per MAVLink spec
```

Comment:
This should be max(1, confidence)

Reply:
Thanks @clydemcqueen 
I opened a follow up pr to fix this. thanks for letting us know!